### PR TITLE
create home view

### PR DESCRIPTION
**Pull request by hieptq-0855**
https://edu-redmine.sun-asterisk.vn/issues/9420

### CODE REVIEW

**Comment by hieptq-0855** [comment]
ready

File: resources/views/student/home.blade.php [hunk 1]
```diff
@@ -1,13 +1,17 @@
...
+@section('title')
+    {{ __('lang.home') }}
+@endsection
+@section('content')
+    <div class="container-fluid p-0 home-background-img">
+        <div class="flex-center position-ref full-height">
+            <div class="content">
+                <div class="title m-b-md text-color">
+                	@if(isset($student))
+                    {{ __('lang.welcome').': '.$student->full_name }}
```
Comment (BunnyPi04) [inline resources/views/student/home.blade.php]: convention

**Comment by hieptq-0855** [comment]
ready

File: resources/views/student/home.blade.php [hunk 2]
```diff
@@ -1,13 +1,17 @@
...
+@section('title')
+    {{ __('lang.home') }}
+@endsection
+@section('content')
+    <div class="container-fluid p-0 home-background-img">
+        <div class="flex-center position-ref full-height">
+            <div class="content">
+                <div class="title m-b-md text-color">
+                    @if(isset($student))
+                        {{ __('lang.welcome').': '.$student->full_name }}
```
Comment (BunnyPi04) [inline resources/views/student/home.blade.php]: convention

**Comment by hieptq-0855** [comment]
ready

**Comment by hieptq-0855** [comment]
ready

**Pull request merged**